src/docfx.website.themes/default/ManagedReference.html.primary.extension.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+exports.preSteps = function (model) {
+  return model;
```

Comment:
As discussed, remain this as user may want to return a new model.

---
In reply to: [84629091](https://github.com/dotnet/docfx/pull/811#discussion_r84629091) [](ancestors = 84629091)